synapse/lib/cell.py

```diff
@@ -3217,7 +3217,7 @@ async def _initCellHive(self):
     async def _initSlabFile(self, path, readonly=False):
         slab = await s_lmdbslab.Slab.anit(path, map_size=SLAB_MAP_SIZE, readonly=readonly)
         slab.addResizeCallback(self.checkFreeSpace)
-        self.onfini(slab)
+        self.onfini(slab.fini)
```

Comment:
This will leak a reference to the slab if it is ever deleted. We'll need to adjust the behavior of onfini() to fix :)